src/server/worldserver/worldserver.conf.dist

```diff
@@ -796,7 +797,7 @@ CharactersPerRealm = 12
 #
 #    HeroicCharactersPerRealm
 #        Description: Limit number of heroic class characters per account on this realm.
-#        Range:       1-10
+#        Range:       1-12
 #        Default:     1
```

Comment:
Shouldn't you change `Default:     1` to `Default:     1`? DKs and DHs are both Heroic Characters. If `Default=1`, shouldn't that prevent players from creating DH if DK is already created and vice-versa?